### PR TITLE
Adds a second rescue RIG to the First Responder's offiice

### PR DESCRIPTION
**Pull request by FayeLovesU**
There are two responders, and only one RIG. The rig is a ton of fun to use, with a slick sprite and a jetpack, leg actuators, etc. However, since there's only one it leads to one FR usually feeling left out, especially if the other FR isn't the sharing sort. It isn't fun and it sucks to be in the normal voidsuit while your partner is in the rig,- like driving a model T alongside the batmobile. Even when you are the one using it, I find that it's a source of worry that you may be hogging it from another FR. 

Machinists can produce these, yes, but they're prohibitively expensive - the same as the CE's hardsuit and the hazard RIG -, and dependant on a multitude of factors such as whether or not mining is bringing in materials, or if research is installing upgrades to reduce costs. I've tried multiple times to get a spare RIG from the machinists in-round, but have not been successful.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
Add a changelog, please. Here's how.
1. Go into the Aurora/html/changelogs file.
2. Copy + Paste 'example.yml'.
3. Rename it to [username]_[commit name]. This isn't standard, but what I usually do.
4. Edit the contents. Remember to change the author.
5. Stage, push and commit.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
what wezzy said above.

**Comment by FayeLovesU** [comment]
changelog done : )

**Review by Llywelwyn — CHANGES_REQUESTED** [review]
the actual changes are all good, but changelog needs to be a new file. right now you've renamed example.yml so it doesn't exist anymore, so there'll probably be some error thrown when the checks are ran

**Comment by FayeLovesU** [comment]
> the actual changes are all good, but changelog needs to be a new file. right now you've renamed example.yml so it doesn't exist anymore, so there'll probably be some error thrown when the checks are ran
> 
> just need to return example.yml as it was and put your changelog in a new copy

Pretty sure I fixed it

**Review by Llywelwyn — APPROVED** [review]

File: html/changelogs/FayeLovesU - 14316.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Added a second rescue RIG to the first responder's quarters."
```
Comment (SleepyGemmy) [inline html/changelogs/FayeLovesU - 14316.yml:41]:
```suggestion
  - maptweak: "Added a second rescue RIG to the first responder's quarters."
```

**Review by SleepyGemmy — APPROVED** [review]
changes addressed. mapping looks fine.

**Review by JohnWildkins — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**